defaults/main.yml

```diff
@@ -4,9 +4,6 @@ nim_waku_cont_image: 'statusteam/nim-waku:{{ nim_waku_cont_tag }}'
 nim_waku_cont_name: 'nim-waku'
 nim_waku_cont_vol: '/docker/{{ nim_waku_cont_name }}'
 
-# Node database path
-nim_waku_node_db_path: '{{ nim_waku_cont_vol }}/data'
```

Comment:
You removed this variable but it's still being used:
https://github.com/status-im/infra-role-nim-waku/blob/9d78bda6a1b392dde4f1042b656958db78f9e0c8/templates/docker-compose.yml.j2#L19-L20
https://github.com/status-im/infra-role-nim-waku/blob/9d78bda6a1b392dde4f1042b656958db78f9e0c8/tasks/container.yml#L4